tests/search/test_solr.py

```diff
@@ -10,6 +10,7 @@
 
 
 class BaseSolrTest(unittest2.TestCase):
+
```

Comment:
On 06/17/2015 05:08 PM, Michael Hrivnak wrote:
> I'm not picky about this, and I don't think pep8 takes a stance on it,
> but I do like consistency. The existing style in this module and others
> is to not use a blank line after a class definition, so I think we
> should stick with that convention.

One day far into the future, I'd like to start using the pep257 module
that is used by a few other projects (like pulp_python and
pulp_openstack). PEP-257 apparently does want this blank line, at least
the pep257 checker complains without it.

Of course, we can add exceptions for the rules we don't care about, and
this does seem fairly arbitrary to me. My personal preference is also
not to have a blank line after a class definition, but to immediately
jump into the docblock.

The great thing about pep257 is that it can enforce the existence of doc
blocks (and I think param definitions too?)

-- 
Randy Barlow